File: HealthCareBillingSystem/HealthCarePaymentTrackerAPi/HealthCareBilling.Model/Model/Patient.cs
using System;

namespace HealthCareBilling.Model
{
    public class Patient
    {
        public int Id { get; set; }
        public string FirstName { get; set; } = string.Empty;
        public string LastName { get; set; } = string.Empty;
        public DateTime DateOfBirth { get; set; }
        public string? Email { get; set; }
        public string? Phone { get; set; }
        public string? Address { get; set; }
        public string? InsuranceProvider { get; set; }
        public string? InsurancePolicyNumber { get; set; }
        public DateTime CreatedAt { get; set; }
        public DateTime UpdatedAt { get; set; }
    }
}

File: HealthCareBillingSystem/src/interfaces/patient.ts
export interface Patient {
  id: number;
  name: string;
  firstName: string;
  lastName: string;
  age: number;
  dateOfBirth: string;
  gender: string;
  phone: string;
  email: string;
  address: string;
  insuranceProvider: string;
  insurancePolicyNumber: string;
}

export interface PatientStats {
  total: number;
  byGender: { [key: string]: number };
  byAgeGroup: { [key: string]: number };
  recent: number;
}
 